### ISSUE TITLE
Wrap jsonrepair as VSCode-Extension?

### ISSUE THREAD

**Issue by manuschillernordlb** [josdejong/jsonrepair#95]
It would be super cool, if we could fix JSON-Files directly within VSCode. Have you considered creating an extension yet?
Would that be something that you would be interesting in, or would it rather be a community contribution? :)

**Comment by josdejong** [comment]
Yes that would be very useful! I do not have time for it though myself any time soon. Anyone interested in implementing a VS Code extension? I would love to see how that works.
